api/v1/ytsaurus_types.go

```diff
@@ -696,6 +696,8 @@ const (
 	UpdateStateWaitingForOpArchiveUpdate          UpdateState = "WaitingForOpArchiveUpdate"
 	UpdateStateWaitingForQTStateUpdatingPrepare   UpdateState = "WaitingForQTStateUpdatingPrepare"
 	UpdateStateWaitingForQTStateUpdate            UpdateState = "WaitingForQTStateUpdate"
+	UpdateStateWaitingForYqlAUpdatingPrepare      UpdateState = "WaitingForYqlAUpdatingPrepare"
```

Comment:
Let's maybe either choose 
`UpdateStateWaitingForYqlAgentUpdatingPrepare` 
or
`UpdateStateWaitingForYqlaUpdatingPrepare`

`...YqlA...` is rather odd abbreviation

Reply:
Changed to `Yqla`